lib/foreman_tasks/engine.rb

```diff
@@ -4,6 +4,11 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_tasks.load_default_settings', :before => :load_config_initializers do
       require_dependency File.expand_path('../../../app/models/setting/foreman_tasks.rb', __FILE__) if (Setting.table_exists? rescue(false))
+      logging = Foreman::Plugin::Logging.new('foreman-tasks')
+      logging.configure(:loggers => {
```

Comment:
Use the plugin API if you can: http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Custom-Plugin-Loggers